### PR TITLE
Update INSTALL.

### PR DESCRIPTION
**Pull request by csukuangfj**
Describe different ways to install k2 with more details.

@qindazhu Please have a review.

### CODE REVIEW

**Comment by qindazhu** [comment]
+2

**Comment by csukuangfj** [comment]
thanks, merging.

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+
+# Installation
+
+k2 supports CUDA as well as CPU. To get the full benefit from k2,
+we recommend that you have installed CUDA on your system before
```
Comment (danpovey) [inline INSTALL.md:5]: Will k2 even run without the CUDA toolkit being installed?

**Comment by csukuangfj** [comment]
I guess so. Will set up an environment to verify it.




Wednesday, 25 November 2020, 11:55 +0800 from notifications@github.com  <notifications@github.com>:

File: INSTALL.md (hunk 1 above)
Comment (csukuangfj) [reply INSTALL.md:5]: Yes, I have tried it locally on a virtual machine without GPU and without CUDA toolkit.
All tests get passed.

The prebuilt wheel package is downloaded from GitHub actions.

**Pull request merged**